libraries/AP_InertialSensor/AP_InertialSensor_MPU6000.cpp

```diff
@@ -252,8 +250,14 @@ AP_InertialSensor_Backend *AP_InertialSensor_MPU6000::probe(AP_InertialSensor &i
 AP_InertialSensor_Backend *AP_InertialSensor_MPU6000::probe(AP_InertialSensor &imu,
                                                             AP_HAL::OwnPtr<AP_HAL::SPIDevice> dev)
 {
-    AP_InertialSensor_MPU6000 *sensor =
-        new AP_InertialSensor_MPU6000(imu, std::move(dev), BUS_TYPE_SPI, false, 0x80);
+    AP_InertialSensor_MPU6000 *sensor;
+
+    dev->set_read_flag(0x80);
+
+    sensor = new AP_InertialSensor_MPU6000(imu,
+                                           std::move(dev),
+                                           BUS_TYPE_SPI,
+                                           false);
```

Comment:
@lucasdemarchi You didn't fix this when merging (and in 9250 too), was it on purpose?